Tests/Resource/ResourceTests.swift

```diff
@@ -24,48 +24,35 @@ extension Todo: ContentMappable {
 extension Todo: StructuredDataRepresentable {
     public var structuredData: StructuredData {
         return [
-            "id": id.map({StructuredData.from($0)}) ?? nil,
-            "title": StructuredData.from(title),
-            "done": StructuredData.from(done)
+            "id": id.map({StructuredData($0)}) ?? nil,
+            "title": StructuredData(title),
+            "done": StructuredData(done)
```

Comment:
Hmm... I'm not a fan of that. Why did we remove `from`? Initializers behave _mostly_ like static functions, but not quite enough I think.